docs/internals/contributing/localizing.txt

```diff
@@ -55,7 +55,7 @@ Formats
 =======
 
 You can also review ``conf/locale/<locale>/formats.py``. This file describes
-the date, time and numbers formatting particularities of your locale. See
+the date, time, and numbers formatting particularities of your locale. See
```

Comment:
This reads alright without a serial comma. Smaller diffs are more mergeable.

Reply:
Reverting...